plugins/generator-addon-1.21.x/addon-1.21.x/item.definition.yaml

```diff
@@ -1,14 +0,0 @@
-templates:
-  - template: item.json.ftl
-    writer: json
-    name: "@SRCROOT/items/@registryname.json"
-  - template: resourcepack/item.json.ftl
-    writer: json
-    name: "@RESROOT/items/@registryname.json"
-
-localizationkeys:
-  - key: item.@modid:@registryname.name
-    mapto: name
-
-field_inclusions: [texture, name, glowCondition, stackSize, damageCount, damageVsEntity, enableMeleeDamage,

```

Comment:
isMeat is still supported, but now as a tag component

Reply:
However, you told me above to undo the switch to format 1.21.50, so do I revert to 1.12 or do I keep this new format. I'm not sure as you're talking about a 1.21.50 change (or do you give me the info for the future PR?)

Reply:
(I'm leaving this comment open, so I remember it when I'll be there.)